miner/algo_common.go

```diff
@@ -455,216 +299,111 @@ func insertPayoutTx(env *environment, sender, receiver common.Address, gas uint6
 	return nil, err
 }
 
-func (envDiff *environmentDiff) commitPayoutTx(amount *big.Int, sender, receiver common.Address, gas uint64, prv *ecdsa.PrivateKey, chData chainData) (*types.Receipt, error) {
```

Comment:
I didn't want to duplicate the payout transaction code for mulit-tx-snapshot

It was necessary to refactor this to avoid duplicates of critical code